### PR TITLE
Entity pulse effect on rejected submissions

### PR DESCRIPTION
**Pull request by NSUWAL123**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation
- [ ] 🧑‍💻 Refactor
- [ ] ✅ Test
- [ ] 🤖 Build or CI
- [ ] ❓ Other (please specify)

## Related Issue
 - Related to #1975

## Describe this PR
- Pulse entity if submission to that entity marked as `hasIssues`
- Remove `rejected` entity submission state, retaining only `approved` and `hasIssues` state

## Screenshots
[Screencast from 17-1-25 03:32:55 अपराह्न +0545.webm](https://github.com/user-attachments/assets/9d454a2f-d276-4ee6-ae96-40348c21741b)

## [optional] What gif best describes this PR or how it makes you feel?
giphy (1)

### CODE REVIEW

**Comment by NSUWAL123** [comment]
Now everything works fine but one small problem due to the current workflow: 
For example, if a submission has been made to an entity and it has been marked as bad then the obvious behavior would be that the entity's color would be red and the highlighting effect would be visible. Now the issue arises when another submission is made to that entity. The entity's color is changed to green but the highlighting effect remains until the submission is marked as `approved` since delete geom record API is triggered on marking the submission as approved.
So, basically in our current workflow, we are not able to delete geometry records once a submission is made to the entity whose previous submission has been marked `hasIssues` leading to the entity color green while having red pulse effect.

image

**Review by spwoodcock — APPROVED** [review]
Lots of changes, two frontend 😅
Good work!

Hopefully we don't have many changes that affect both the React and Svelte frontends at once.

Thanks for commenting on the case of two submissions too 👍 we can return to that in the future if needed
